mscore/playpanel.cpp

```diff
@@ -350,7 +350,7 @@ void PlayPanel::volLabel()
       if (vol == MUTE)
             vol = -80.0;
       else
-            vol = ((n * std::log10(vol)) - n);
+            vol = ((N * std::log10(vol)) - N);
```

Comment:
That formula looks somewhat strange, if I don't miss something this doesn't look like a proper decibel definition although the difference between the definitions seems to be only about 3 dB (if other places in the code use the same reference volume value). Of course, this has nothing to do with compiler warnings but it should probably be checked whether this definition matches other formulae used within MuseScore code.

Reply:
What would the correct formula be then?